googleapiclient/sample_tools.py

```diff
@@ -72,7 +72,7 @@ def init(argv, name, version, doc, filename, scope=None, parents=[], discovery_f
   # application, including client_id and client_secret, which are found
   # on the API Access tab on the Google APIs
   # Console <http://code.google.com/apis/console>.
-  client_secrets = os.path.join(os.path.dirname(filename),
+  client_secrets = os.path.join(os.path.dirname(filename), os.getcwd(),
```

Comment:
I don't think this works the way you expect. What's the case that this is supposed to fix?